src/relay/backend/graph_plan_memory.cc

```diff
@@ -209,15 +209,18 @@ class StorageAllocator : public StorageAllocaBaseVisitor {
     for (const auto& kv : token_map_) {
       std::vector<Integer> storage_ids;
       std::vector<Integer> device_types;
+      std::vector<Integer> sid_sizes_byte;
```

Comment:
i'm not sure this is exercised anywhere in unit test. can you add one? right now we just assert on the len() but not the content